internal/presenters/templates/local_finding.tmpl

```diff
@@ -22,7 +22,7 @@
 
 {{define "summary"}}{{ "Test Summary" | bold }}
 
-  Organization:      {{.Summary.Org}}
+  Organization:      {{ getValueFromConfig "internal_org_slug" }}
```

Comment:
**question:** Do we want to normalise use of `internal_` prefixed properites within the templates if we are aiming for this to become an external interface? 

Or do I misunderstand the purpose of "internal_org_slug"?

Reply:
Sorry, this change can actually undone. It was meant for testing reading from the config. There is an explicit ticket to implement this.

Reply:
Reverted!

Reply:
Thats what I thought :)